### ISSUE TITLE
Ontology import fails occasionally 

### ISSUE THREAD

**Issue by pplettner** [althonos/pronto#9]
Hi, thanks for fixing the inline comments bug so quickly. I have another issue to report, but this is harder to pin down since it doesn't happen all the time (it happens maybe 10-20% of the time?)

If I import the Uberon ontology:

`uberon_ontology = Ontology(path='http://purl.obolibrary.org/obo/uberon.owl', imports=False)`

I sometimes get this exception and it fails:

```
Traceback (most recent call last):
  File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/parser/owl.py", line 441, in _classify
    new_term[owl_to_obo[k]] = rawterm[k]['data']
KeyError: 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./import_ontology.py", line 61, in <module>
    uberon_ontology = Ontology(path='http://purl.obolibrary.org/obo/uberon.owl', imports=False)
  File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/ontology.py", line 80, in __init__
    self.parse(handle, parser)
  File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/ontology.py", line 197, in parse
    self.meta, self.terms, self.imports = p.parse(stream)
  File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/parser/owl.py", line 368, in parse
    terms = cls._classify(_rawterms)
  File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/parser/owl.py", line 443, in _classify
    new_term[k] = rawterm[k]['data']
KeyError: 'data'
```
The specific line in the ontology that makes this fail is this:

`<obo:UBPROP_0000008 rdf:datatype="http://www.w3.org/2001/XMLSchema#string"></obo:UBPROP_0000008>`

which makes:
`k=UBPROP_0000008`
`rawterms[k]={'datatype': 'http://www.w3.org/2001/XMLSchema#string'}`

It's strange because it can succeed if I run it again, but not always. Thanks in advance.

**Comment by althonos** [comment]
Hi, thanks for the report.
*pronto* uses two different parsers for `owl` files, and only one of those works fine with uberon.
As a hotfix, you can add the keyword argument `parser="OwlXMLTreeParser"` to use the one that works.

I'll see why the target parser crashes.

**Comment by althonos** [comment]
Hi @pplettner : this should be fixed with latest release (`v0.9.0`), at least the following code:
```python
import pronto
ub=pronto.Ontology('http://purl.obolibrary.org/obo/uberon.owl')
```
does not raise any error when I tried it.
